### PR TITLE
Add PolyMC.app v1.1.1

### PR DESCRIPTION
**Pull request by kthchew**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [X] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [X] `brew audit --cask <cask>` is error-free.
- [X] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [X] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [X] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [X] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [X] `brew audit --new-cask <cask>` worked successfully.
- [X] `brew install --cask <cask>` worked successfully.
- [X] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/polymc.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  url "https://github.com/PolyMC/PolyMC/releases/download/#{version}/PolyMC-macOS-#{version}.tar.gz",
+      verified: "github.com/PolyMC/PolyMC/"
+  name "PolyMC"
+  desc "Minecraft launcher"
+  homepage "https://polymc.org/"
+
+  livecheck do
+    url :url
+  end
+
```
Comment (bevanjkay) [inline Casks/polymc.rb:14]: @kthchew This can be omitted as it will be automatically inferred from the url.

File: Casks/polymc.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  depends_on macos: ">= :sierra"
+
+  app "PolyMC.app"
+
+  zap trash: [
+    "~/Library/Application Support/PolyMC/metacache",
+    "~/Library/Application Support/PolyMC/PolyMC-*.log",
+    "~/Library/Application Support/PolyMC/polymc.cfg",
```
Comment (bevanjkay) [inline Casks/polymc.rb:22]: @kthchew Can we trash `~/Library/Application Support/PolyMC` instead?

**Review by bevanjkay — CHANGES_REQUESTED** [review]

File: Casks/polymc.rb (hunk 1 above)
Comment (kthchew) [reply Casks/polymc.rb]: Done, thanks.

File: Casks/polymc.rb (hunk 2 above)
Comment (kthchew) [reply Casks/polymc.rb:18]: I don't think so. That directory contains user instances (game data, world files, etc.) in `~/Library/Application Support/PolyMC/instances` that are directly created by the user. AFAIK those shouldn't be deleted by a zap. There's also other directories in there that usually contain auto-generated files, but may have been modified or created by a user manually, so I wanted to be safe and not delete them.

Comment (kthchew) [reply Casks/polymc.rb:18]: For reference when I open PolyMC for the first time and then create an instance, I see the following in `~/Library/Application Support/PolyMC`:

```
icons (directory, contains image files a user places there)
instances (directory, contains user data)
PolyMC-0.log (file)
polymc.cfg (file)
themes (directory, contains a default theme and/or user customizations)
translations (directory)
meta (directory, contains info about game/library versions but these can be edited by the user if they need something custom)
metacache (file)
```

**Review by bevanjkay — APPROVED** [review]

**Comment by bevanjkay** [comment]
Thank you @kthchew

**Pull request merged**